spec/acceptance/native/backup_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'spec_helper_acceptance'
```

Comment:
The examples in spec/acceptance/native/ test the native types directly. They're modified originals.